float64 x
float64 y
std_msgs/String user
---
bool success
